inst/resources/js/autoscaling_iframe.js

```diff
@@ -0,0 +1,139 @@
+// An auto-scaling iframe
```

Comment:
FWIW I used iFrame Resizer in Distill and was quite happy with it: http://davidjbradshaw.github.io/iframe-resizer/

Reply:
I was unaware of iFrame Resizer. This looks awesome! I will give it a try. Thanks!